app/uk/gov/hmrc/agentsubscriptionfrontend/connectors/AuthenticatorConnector.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2017 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.agentsubscriptionfrontend.connectors
+
+import java.net.URL
+import javax.inject.{Inject, Named}
+
+import play.api.Logger
+import uk.gov.hmrc.play.http.{HeaderCarrier, HttpGet, HttpPost, HttpResponse}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+class AuthenticatorConnector @Inject()(@Named("government-gateway-authentication-baseUrl") baseUrl: URL, http: HttpGet with HttpPost) {
+
+  def refreshEnrolments(implicit hc: HeaderCarrier, ec: ExecutionContext): Future[HttpResponse] = {
+    http.POSTEmpty(s"$baseUrl/government-gateway-authentication/refresh-profile") map { response =>
```

Comment:
I am guessing POSTEmpty is a new thing with http-verbs?  Also we are meant to be using MdcLoggingExecutionContext  instead of ExecutionContext ;)